ReactAndroid/src/main/java/com/facebook/react/uimanager/Spacing.java

```diff
@@ -55,10 +55,23 @@ public class Spacing {
    * Spacing type that represents the block start direction (top). E.g. {@code marginBlockStart}.
    */
   public static final int BLOCK_START = 11;
+  /** Spacing type that represents inline directions (left, right). E.g. {@code marginInline}. */
+  public static final int INLINE = 12;
+  /**
+   * Spacing type that represents the inline end direction (right in left-to-right, left in
+   * right-to-left). E.g. {@code marginInlineEnd}.
+   */
+  public static final int INLINE_END = 13;
+  /**
+   * Spacing type that represents the inline start direction (left in left-to-right, right in
+   * right-to-left). E.g. {@code marginInlineStart}.
+   */
+  public static final int INLINE_START = 14;
```

Comment:
Same comment as the `inline PR`, it could create confusion to have `Spacing.END` not be the last enum here. As we are adding, I think we should update this. If there is existing code relying on `Spacing.END` which only needs the ones before it, we should add a new well-known enum where `Spacing.END` used to be.